code/Controllers/CMSMain.php

```diff
@@ -1038,56 +1038,93 @@ public function getBreadcrumbsBackLink()
             ->Link;
     }
 
+    /**
+     * Generate a breadcrumb based on the page currently being viewed.
+     * @param bool $unlinked Wheter to include links
+     * @return ArrayList<ArrayData>
+     */
```

Comment:
The param and return types are inherited from `LeftAndMain` so no need to add those.
The phpdoc comment is in the wrong place - should be added to `LeftAndMain`.
```suggestion
```

Reply:
Done